cloud/google/super_duper_service.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "super-duper-service",
+  "apiVersion": "v1beta1",
+  "kind": "Service",
+  "selector": {
+    "name": "log-node"
+  },
+  "containerPort": "super-duper",
+  "protocol": "TCP",
+  "port": 80,
+  "createExternalLoadBalancer": true
```

Comment:
There's some docage on the Nginx site about this, basically the free one backs off a node if it gets an error back while serving a user request, the payware version does configurable OOB health checking.